functions/New-DbaXESmartCsvWriter.ps1

```diff
@@ -0,0 +1,95 @@
+function New-DbaXESmartCsvWriter {
+    <#
+        .SYNOPSIS
+            This Response type is used to write Extended Events to a CSV file.
+
+        .DESCRIPTION
+            This Response type is used to write Extended Events to a CSV file.
+
+        .PARAMETER OutputFile
+            Specifies the path to the output CSV file.
+
+        .PARAMETER OverWrite
```

Comment:
Minor case update. The parameter name is showing as `Overwrite` in param block, just need to update this one to match.